src/ert/scheduler/openpbs_driver.py

```diff
@@ -93,12 +141,19 @@ async def submit(
             *args,
         ]
         logger.debug(f"Submitting to PBS with command {shlex.join(qsub_with_args)}")
-        process = await asyncio.create_subprocess_exec(
-            *qsub_with_args,
-            stdout=PIPE,
+
+        # The failed exit_codes from qsub command
+        # 171, Invalid credential
+        # 183, Premature end of message
+        # 162, Connection refused
+        job_success, job_message = await self._execute_with_retry(
```

Comment:
I will claim `job_*` are misleading variable names here, since "job" is the Entire Thing, but this is only about the qsub process.